### PR TITLE
Fix deprecation warnings

### PR DESCRIPTION
**Pull request by deining**
### Prerequisites

Put an `x` into the box(es) that apply:

- [x] This pull request fixes warnings.
- [ ] This pull request adds a feature.
- [ ] This pull request introduces breaking change.

### Description

This PR fixes warnings emitted by hugo.
It raises the min required version for the theme to 0.124.0, I hope this is o.k. for you.

#### Contributors

- [x] Add yourself to `CONTRIBUTORS.md` if you aren't on it already

### CODE REVIEW

File: hugo.toml [hunk 1]
```diff
@@ -2,4 +2,4 @@ baseURL = "https://example.com/"
 
 [module]
 [module.hugoVersion]
-min = "0.77.0"
\ No newline at end of file
+min = "0.124.0"
```
Comment (luizdepra) [inline hugo.toml:5]: I need to check if this file is really needed. I think not.
Do you know anything about it?

Comment (luizdepra) [reply hugo.toml:5]: Anyway, could you change the `min_version` value from `theme.toml` too?

Comment (deining) [reply hugo.toml]: > Anyway, could you change the `min_version` value from `theme.toml` too?

Changed now.

**Review by luizdepra — APPROVED** [review]
Thank you. :D

**Pull request merged**